docs/source/fitting/fitfunctions/IkedaCarpenterPV.rst

```diff
@@ -11,20 +11,40 @@ Description
+
+The parameters used above are defined as (from Panel 13 of Ref [3]):
+
++------------------------------------+------------------------------------------------------------------------+--------------------------------------------------------------+------------------------------------------------------+---------------------------------------------------------------------+
+| .. math:: k = 0.05                 | .. math:: z_s = -\alpha dt + i\frac{1}{2} \alpha \gamma                | .. math:: u = \frac{1}{2} \alpha^- (\alpha^- \sigma^2 - 2dt) | .. math:: N = \frac{1}{4} \alpha \frac{(1-k^2)}{k^2} | .. math:: y_u = \frac{ (\alpha^- \sigma^2 - dt) }{\sqrt{2\sigma^2}} |
+| .. math:: \alpha^- = \alpha(1 - k) | .. math:: z_u = -\alpha^- dt + i\frac{1}{2} \alpha^- \gamma = (1-k)z_s | .. math:: v = \frac{1}{2} \alpha^+ (\alpha^+ \sigma^2 - 2dt) | .. math:: N_u = 1 - R \frac{\alpha^-}{x}             | .. math:: y_v = \frac{ (\alpha^+ \sigma^2 - dt) }{\sqrt{2\sigma^2}} |
+| .. math:: \alpha^+ = \alpha(1 + k) | .. math:: z_v = -\alpha^+ dt + i\frac{1}{2} \alpha^+ \gamma = (1+k)z_s | .. math:: s = \frac{1}{2} \alpha (\alpha \sigma^2 - 2dt)     | .. math:: N_v = 1 - R \frac{\alpha^+}{z}             | .. math:: y_s = \frac{ (\alpha \sigma^2 - dt) }{\sqrt{2\sigma^2}}   |
+| .. math:: x = \alpha^- - \beta     | .. math:: z_r = -\beta dt + i\frac{1}{2} \beta \gamma                  | .. math:: r = \frac{1}{2} \beta (\beta \sigma^2 - 2dt)       | .. math:: N_s = -2(1 - R\frac{\alpha}{y})            | .. math:: y_r = \frac{ (\beta \sigma^2 - dt) }{\sqrt{2\sigma^2}}    |
+| .. math:: y = \alpha - \beta       |                                                                        |                                                              | .. math:: N_r = 2R\alpha^2\beta \frac{k^2}{xyz}      |                                                                     |
+| .. math:: z = \alpha^+ - \beta     |                                                                        |                                                              |                                                      |                                                                     |
++------------------------------------+------------------------------------------------------------------------+--------------------------------------------------------------+------------------------------------------------------+---------------------------------------------------------------------+
+
+where :math:`dt = T_i - T_h` is the shift in microseconds with respect to the Bragg position,
+:math:`\alpha` and :math:`\beta` are the fast and slow neutron
+decay constants respectively, and :math:`R` is a maxing coefficient that
+relates to the moderator temperature.
 :math:`\alpha` and :math:`R` are further modelled to depend on
 wavelength and using the notation in the Fullprof manual (Ref [2]) the
```

Comment:
Just a tiny change needed here,

change

`in the Fullprof manual (Ref [2]) the`

to 

`in the Fullprof manual (Ref [2]). The`